deid/dicom/parser.py

```diff
@@ -167,7 +167,7 @@ def get_nested_field(self, field, return_parent=False):
 
         if return_parent:
             return parent, desired
-        return desired
+        return parent[desired]
```

Comment:
Sorry for the delay responding... this took a bit to research:

**Level 1 Fields (Fields not nested within Sequences)**
It won't return a key error because by the time we get to this point, we're ensured that the tag exists in the header because expand_field_expression either expanded the expander into a list of tags, expanded the values/fields expression, or in the case of a single field being specified by the recipe, returned a dictionary containing the single field.  We don't attempt to run the action if expand_field_expression returns an empty list - no header fields identified.

That said, get_nested_field is used in only two places:

1. delete_field
2. blank_field

In blank_field, we expect that the value returned from get_nested_field is a DataElement.  If we return desired, the value is a BaseTag, while delete utilizes the parent with the Tag index to delete the field.  If we want to keep the return type of "desired" consistent (which I actually prefer), see what I just committed. 

**Nested Fields (Sequences)**
I think we have a bigger problem here. When attempting to BLANK a field nested within a sequence (example: ```(0040, 0275)__0__(0008, 0050)```), the index between the two tags is causing us to return the tuple from expand_field_expression with parent = None.

For our efforts we generally just remove sequences, so handling this isn't a high priority, and honestly when I think it is handled we need to look at sequences as a whole.  Would you be okay with punting on this portion of it for now?

Reply:
Sure! If the problem exists in the current code, then this PR still makes the base better (and we can work on that next). Could you open an issue describing it in detail (likely copy pasting most of the above) so we don't forget?

And so I don't forget like last time - let's bump the version and add a note to the changelog! if you want to release this right away vs. wait for the fix for the above, it's up to you. Let me know!

Reply:
Version and changelog are already bumped. :)  

I got a little sidetracked today.  Discovered that I was looking at an old version of the DICOM spec and was missing some VRs from my unit tests (hence today's update to the deid-data).  I'm pretty confident now that everything is covered.  Just committed some additional changes so that now all VRs are allowed for BLANK and covered by the unit tests as well.